src/assets/css/footer.scss

```diff
@@ -0,0 +1,206 @@
+//Prueba para tener contenido y ver el footer al final
+body > p {
+  background-color: $moramia-blue;
+  height: 100vw;
+}
+* {
+  box-sizing: border-box;
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  text-decoration: none;
+}
```

Comment:
avoid the use of `*` it's not performant and some properties don't apply, like `list-style` applies for lists only.

Reply:
Done